receiver/hostmetricsreceiver/internal/scraper/systemscraper/factory_test.go

```diff
@@ -0,0 +1,20 @@
+package systemscraper
+
+import (
+	"context"
+	"testing"
+
+	"github.com/tj/assert"
```

Comment:
shouldn't this be `testify/assert`?

Reply:
Oops thanks for catching that, `goimports` decided something totally random for me here for some reason.

Reply:
Fixed in [fc04bce](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36123/commits/fc04bce64eda7b8f0bb1510550c002ac78fba649)